CMSgov/mint-app
# [feature: EASI-3923] Conditional relation translation updates

# EASI-3923


## Changes and Description

This is a high level review of the previous PRs for readonly conditional relationship updates

https://github.com/CMSgov/mint-app/pull/956
https://github.com/CMSgov/mint-app/pull/959
https://github.com/CMSgov/mint-app/pull/961
https://github.com/CMSgov/mint-app/pull/968
https://github.com/CMSgov/mint-app/pull/971




## How to test this change

Verify Figma matches, notes from previous PRs considered
[Figma](https://www.figma.com/file/6tkIhfNfcsKEAkqXo1qvXH/MINT---Read-Only-View?type=design&node-id=4399-18020&mode=design&t=xb1fW9UwQAgs53Gf-0)

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
